src/entity/commit.py

```diff
@@ -61,6 +61,10 @@ def detect_language(file_path):
         return 'Shell'
     if extension == 'swift':
         return 'Swift'
+    if extension == 'ts':
```

Comment:
We prefer the if or. As it is done with HTML or Haskell. Like this, each language has only 1 if statement.